metaflow/plugins/package/package_decorator.py

```diff
@@ -0,0 +1,70 @@
+import os
```

Comment:
This now seems to be an empty hull and a lot of code can go away. I wonder if we can also move the env-vars part directly to runtime.py and get rid of this extra decorator.